### ISSUE TITLE
What Makes You Different?

### ISSUE THREAD

**Issue by zmon** [zmon/red-tape#4]
### What's your Unique Value Proposition?

Fill in Box 3, the "Unique Value Proposition."

What is a UVP? It's the promise you're making to your customer that's compelling enough for them to jump on board.

Ask yourself "If I solve this problem (Box 1) `[by doing this]` for you (customer from Box 2), is that compelling?". Whatever the `[by doing this]` is the Unique Value Proposition you're proposing.

Don't forget to fill out the `High Level Concept`, that's a clear way to describe what you are. For Uber this would be "Order a taxi with your phone."

In short, you want to make Conan doing this.

![Conan](https://media1.giphy.com/media/OaBqrGWlQPQru/200.gif)
